src/pages/search/project_search/ProjectSearch.tsx

```diff
@@ -24,14 +25,22 @@ export const ProjectSearch = () => {
   if (!projectSearchQueryParams.get(SearchParam.Results)) {
```

Comment:
Trengs fortsatt `.` over her for prosjektsøk? mener at noen fjernet det kravet for en liten tid tilbake, men ikke sikker

Reply:
Trodde den måte være der for at `titleQuery` skulle fungere, men det funker uten👌